docs/readthedocs/models/reference_system.md

```diff
@@ -17,14 +17,11 @@ The reference system is built up by specifying the included voltage levels. The
 
 To configure the reference system, the voltage levels listed in the table above must be selected and integrated into the reference system with the nominal apparent power (sNom), the nominal voltage (vNom), and the individual voltage level configuration (voltLvls) according to the example below. Each voltage level is composed of the identifier and the nominal voltage.
 
-`simona.gridConfig.refSystems = [`
-
-`{sNom = "100 kVA", vNom = "0.4 kV", voltLvls = [{id = "NS", vNom = "0.4 kV"}]},`
-
-`{sNom = "60 MVA", vNom = "20 kV", voltLvls = [{id = "MS", vNom = "20 kV"}]},`
-
-`{sNom = "600 MVA", vNom = "110 kV", voltLvls = [{id = "HS", vNom = "110 kV"}]},`
-
-`{sNom = "1000 MVA", vNom = "380 kV", voltLvls = [{id = "HoeS", vNom = "380 kV"}]}`
-
-`]`
+```
+simona.gridConfig.refSystems = [
+    {sNom = "100 kVA", vNom = "0.4 kV", voltLvls = [{id = "NS", vNom = "0.4 kV"}]},
```

Comment:
Should the identifier of the voltage levels be adapted?

Reply:
ATM we leave it as it is. I'll take this into our group and if there is a vote for changing it we will create a new issue.